### DISCUSSION TITLE
Docker SetUp + HTPPS + privkey.pem not found

### DISCUSSION THREAD

**Discussion by noteurt** [TriliumNext/Trilium#2733] [Q&A] [answered]
Hello guys,

i am a huge fan of trilium, i set up a docker image on my vps. The problems are the certificates, i have them in my trilium-data directory, but docker logs always:

```
Error: ENOENT: no such file or directory, open '/opt/trilium-data/certs/privkey.pem'
    at Object.openSync (fs.js:497:3)
    at Object.readFileSync (fs.js:393:35)
    at startTrilium (/usr/src/app/src/www:57:21)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/opt/trilium-data/certs/privkey.pem'
}
```

The directory certs dir has the following permission: 
`drwxr-xr-x 2` 

and the fullchain and the privkey have the following permissions: 
`-r--r--r--`

The odd thing is that the fullchain doesn't throw an error.

Thanks in advance for your help

Greetings noteurt

**Comment by sigaloid** [comment, marked as answer]
Inside the docker container, the file path for trilium-data is accessible frm `/home/node/trilium-data`, not `/opt/trilium-data/`. Try modifying the path to that.

**Reply by noteurt** [reply]
Thank you!
